### PR TITLE
Changes related to the next Meilisearch release (v0.29.0)

### PR DESCRIPTION
**Pull request by alallema**
Related to this issue: https://github.com/meilisearch/integration-guides/issues/211

This PR:
- gathers the changes related to the next Meilisearch release (v0.29.0) so that this package is ready when the official release is out.
- should pass the tests against the [latest pre-release of Meilisearch](https://github.com/meilisearch/meilisearch/releases).
- might eventually contain test failures until the Meilisearch v0.29.0 is out.

⚠️ This PR should NOT be merged until the next release of Meilisearch (v0.29.0) is out.

_This PR is auto-generated for the [pre-release week](https://github.com/meilisearch/integration-guides/blob/master/guides/pre-release-week.md) purpose._

### CODE REVIEW

File: src/Meilisearch/Key.cs [hunk 1]
```diff
@@ -83,6 +83,10 @@ public enum KeyAction
         /// <summary>
         /// Provides access to the add documents and update documents endpoints.
         /// </summary>
+        DocumentsAll,
+        /// <summary>
+        /// Provides access to all documents endpoints.
+        /// </summary>
         DocumentsAdd,
```
Comment (bidoubiwa) [inline src/Meilisearch/Key.cs]: I think the comments above these two are mixed up

File: src/Meilisearch/Key.cs [hunk 2]
```diff
@@ -95,6 +99,10 @@ public enum KeyAction
         /// <summary>
         /// Provides access to the create index endpoint.
         /// </summary>
+        IndexesAll,
+        /// <summary>
+        /// Provides access to a indexes endpoint.
+        /// </summary>
```
Comment (bidoubiwa) [inline src/Meilisearch/Key.cs]: Same for here, also I'm not sure about the description

File: src/Meilisearch/Key.cs [hunk 3]
```diff
@@ -117,6 +125,10 @@ public enum KeyAction
         /// <summary>
         /// Provides access to the get settings endpoint and equivalents for all subroutes.
         /// </summary>
+        SettingsAll,
+        /// <summary>
+        /// Provides access to all settings endpoints and equivalents for all subroutes.
+        /// </summary>
```
Comment (bidoubiwa) [inline src/Meilisearch/Key.cs]: here as well, sorry if I misunderstood the order of the comments

File: src/Meilisearch/Key.cs [hunk 4]
```diff
@@ -138,6 +150,10 @@ public enum KeyAction
         /// <summary>
         /// Provides access to the get all keys endpoint.
         /// </summary>
+        KeysAll,
+        /// <summary>
+        /// Provides access to all keys endpoint.
+        /// </summary>
```
Comment (bidoubiwa) [inline src/Meilisearch/Key.cs]: Here as well

**Review by bidoubiwa — DISMISSED** [review]
LGTM 🔥

**Review by bidoubiwa — APPROVED** [review]
LGTM 🔥

**Comment by alallema** [comment]
bors merge

**Comment by alallema** [comment]
Thanks again for contributing with Meilisearch :heart:
If you are participating in [Hacktoberfest](https://blog.meilisearch.com/contribute-hacktoberfest-2022/), and you would like to receive a small gift from Meilisearch too, don't forget to fill [this form](https://meilisearch.typeform.com/to/idt5qugY).

**Pull request merged**